index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Website</title>
+    <link rel="stylesheet" href="/styles.css">
```

Comment:
`href` link not working cause you are missing a dot before the slash and because of that, I can't see your style try to add the dot like that `href="./styles.css"`

Reply:
i will do